PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+### Before you submit this, make sure you have:
+
+ - [ ] Ran format.py to ensure code is formatted
+ - [ ] Written tests if you added new features to WPILib
+ - [ ] Modified tests if you changed behavior
+ - [ ] Made corresponding changes in *both* Java and C++
+ - [ ] Made corresponding changes in simulation (again, Java and C++)
+
+### Things that will make it less likely for your PR (Pull Request) to be accepted:
+
+ - [ ] Breaking API. This means changing function names or public variable names.
+ - [ ] Combining stylistic changes with functional changes. Break it into two smaller PRs.
+ - [ ] Large PRs. If you change 500 lines, it's going to take more work to review.
+       Keep your PRs small and easy to read.
+ - [ ] Unessecary changes. They distract reviewers, and make it take longer.
```

Comment:
Since `make it take longer` is a dependent clause, there should not be a comma before the "and".